.devcontainer/Install-Dependencies.ps1

```diff
@@ -0,0 +1,63 @@
+$ErrorActionPreference = "STOP"
+
+# Bootstrap Requirements
+Install-Module -Name Requirements -Force
+
+Import-Module -Name Requirements
+
+$basecli = @(
```

Comment:
This Name, Describe, Test, Set is our primary building block for configuration of a docker image.